docs/storage/gridftp.md

```diff
@@ -1,168 +1,123 @@
 
-Review `/etc/edg-mkgridmap.conf` to make sure that it has all VOs that you are interested in and also to comment out any VOs that you do not wish to support.
+#### Authentication with edg-mkgridmap
 
-``` rootscreen
-vi /etc/edg-mkgridmap.conf
-```
+!!! warning
+    As of the June 2017 release of OSG 3.4.0, this software is officially deprecated. Support is scheduled to end as of June 2018.
 
-This utility `edg-mkgridmap` runs as a cronjob `/etc/cron.d/edg-mkgridmap-cron` (by default every 6 hours). You can also run `edg-mkgridmap` manually to see that it generates `/etc/grid-security/grid-mapfile`.
+By default, GridFTP uses a gridmap file, found in `/etc/grid-security/grid-mapfile`. This is the file generated by `edg-mkgridmap` if you follow the default [install instructions](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Edg-mkgridmap).  No further configuration is needed.
 
-``` rootscreen
-edg-mkgridmap
-```
+#### Authentication with GUMS
 
-Then, you can enable/start the service.
+!!! warning
+    As of the June 2017 release of OSG 3.4.0, this software is officially deprecated. Support is scheduled to end as of June 2018.
 
-``` rootscreen
-/sbin/service edg-mkgridmap start
-/sbin/chkconfig edg-mkgridmap on
-```
+If you want to use GUMS security (recommended), you will need to enable it using the following steps:
 
-You can read more on this page: [edg\_mkgridmap (on the CE)](Documentation/Release3.InstallComputeElement#5_2_Using_edg_mkgridmap_for_auth) <span class="twiki-macro ENDSECTION">Gridmap</span>
+1.  Edit `/etc/grid-security/gsi-authz.conf` and uncomment the globus callout.
 
-—\#\#\# Configuring GUMS support
+        :::text
+        globus_mapping liblcas_lcmaps_gt4_mapping.so lcmaps_callout
 
-<span class="twiki-macro STARTSECTION">Gums</span> By default, GridFTP uses a gridmap file, found in `/etc/grid-security/gridmap-file`. If you want to use GUMS security (recommended), you will need to enable it using the following steps:
+    Note that this used to be the full path to the library (`/usr/lib64` or `/usr/lib`), but now we rely on the linker for proper resolution in this file.
 
-First, edit `/etc/grid-security/gsi-authz.conf` and uncomment the globus callout.
+2.  Edit `/etc/lcmaps.db` to include your service endpoint:
 
-``` file
-globus_mapping liblcas_lcmaps_gt4_mapping.so lcmaps_callout
-```
+        :::text
+        ...
+        gumsclient = "lcmaps_gums_client.mod"
+                     "-resourcetype ce"
+                     "-actiontype execute-now"
+                     "-capath /etc/grid-security/certificates"
+                     "-cert   /etc/grid-security/hostcert.pem"
+                     "-key    /etc/grid-security/hostkey.pem"
+                     "--cert-owner root"
+        # Change this URL to your GUMS server
+                     "--endpoint https://%RED%gums.example.com:8443%ENDCOLOR%/gums/services/GUMSXACMLAuthorizationServicePort"
```

Comment:
That's because we disabled it in the technology docs because it was messing with Tim's JIRA links in the meeting notes:

https://github.com/opensciencegrid/technology/commit/7275ec52547a6fa454da8d679b2e4b27ef43977e